src/vscode-dts/vscode.proposed.notebookVariableProvider.d.ts

```diff
@@ -9,7 +9,10 @@ declare module 'vscode' {
 		variableProvider?: NotebookVariableProvider;
 	}
 
-	type VariablesRequestKind = 'named' | 'indexed';
```

Comment:
we need to export this new enum to the api instance.

Reply:
is that done in extHost.api.impl.ts?